docs/schemas-tests/XR-yaml-first/yaml-first-complex-literal-data.yaml

```diff
@@ -0,0 +1,68 @@
+# The following three documents are all equivalent
```

Comment:
These are great indications of the pitfalls here, and why we aren't trying to make yaml produce PFx (or the reverse).